opencensus/stats/aggregation_data.py

```diff
@@ -123,13 +128,14 @@ def __init__(self,
             bounds = []
 
         if counts_per_bucket is None:
-            counts_per_bucket = []
-            bucket_size = len(bounds) + 1
-            for i in range(bucket_size):
-                counts_per_bucket.append(0)
+            counts_per_bucket = [0 for ii in range(len(bounds) + 1)]
+        elif len(counts_per_bucket) != len(bounds) + 1:
+            raise ValueError("counts_per_bucket length does not match bounds "
```

Comment:
I think we can change to ```"counts_per_bucket has X fields, which is more than the Y buckets allowed."``` where, X:len(counts_per_bucket) and Y: len(bounds) + 1. What do you think?

Reply:
One benefit of fixed error messages is that you can grep for the exact text, but I'm happy with either here.